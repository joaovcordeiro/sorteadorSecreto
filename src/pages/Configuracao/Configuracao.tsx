import Formulario from "../../componentes/Formulario/Formulario";
import ListaParticipantes from "../../componentes/ListaParticipantes/ListaParticipantes";
import Rodape from "../../componentes/Rodape/Rodape";

export default function Configuracao() {
  return (
    <section>
      <Formulario />
      <ListaParticipantes />
      <Rodape />
    </section>
  );
}
