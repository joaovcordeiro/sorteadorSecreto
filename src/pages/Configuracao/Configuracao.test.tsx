import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Configuracao from "./Configuracao";

const mockNavigate = jest.fn();

jest.mock("react-router-dom", () => ({
  useNavigate: () => mockNavigate,
}));

describe("a pagina de configuracao", () => {
  test("deve ser renderizda corretamente", () => {
    const { container } = render(
      <RecoilRoot>
        <Configuracao />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
