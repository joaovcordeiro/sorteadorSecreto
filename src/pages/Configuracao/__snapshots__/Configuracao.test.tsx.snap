// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuracao deve ser renderizda corretamente 1`] = `
<div>
  <section>
    <form
      class="sc-bcXHqe"
    >
      <div
        class="sc-gswNZR kjdOqP"
      >
        <input
          placeholder="Insira os nomes dos participantes"
          type="text"
          value=""
        />
        <button
          disabled=""
        >
          Adicionar
        </button>
      </div>
      
    </form>
    <ul />
    <footer
      class="sc-hLBbgP brfTnX"
    >
      <button
        class="sc-eDvSVe ksRRBT"
        disabled=""
      >
        Iniciar brincadeira
      </button>
      <img
        alt="Sacolas de compras"
        src="/imagens/sacolas.png"
      />
    </footer>
  </section>
</div>
`;
